### ISSUE TITLE
Node-Red version

### ISSUE THREAD

**Issue by JohnMayer01** [EdgeVerve/loopback-connector-nodes-for-Node-RED#1]
Hi,
would u mind upgrading your version dependencies of node-red to  "node-red": "^0.14.6"

Best

**Comment by sachinhub** [comment]
@JohnMayer01 the current depenency version "node-red": "^0.13.2" should also pick up 0.14.6 or anything above it. Any particular issue you are facing with the current version in package.json?

**Comment by JohnMayer01** [comment]
Yes, as it only picks up everything in 13.x but not 14.x. Please try it out . You package is downgrading my 14.x npm dependency to 13.x ....

**Comment by pktippa** [comment]
@JohnMayer01 the "node-red" is a devDependency, and it is pointing to latest node-red git, please let us know if you still face the issue, if not we can close this issue.
